--- a/owntone-server/src/transcode.c
+++ b/owntone-server/src/transcode.c
@@ -1442,7 +1442,9 @@
   // values into the the next packet's side data (as AV_FRAME_DATA_REPLAYGAIN),
   // which has the effect that a volume replaygain filter works. Note that
   // ffmpeg itself uses another method in process_input() in ffmpeg.c.
+#if LIBAVFORMAT_VERSION_MAJOR < 60  
   av_format_inject_global_side_data(ctx->ifmt_ctx);
+#endif
 
   ret = avformat_find_stream_info(ctx->ifmt_ctx, NULL);
   if (ret < 0)
